file-native/src/main/java/org/ballerinalang/stdlib/file/utils/ModuleUtils.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.stdlib.file.utils;
+
+import io.ballerina.runtime.api.Environment;
+import io.ballerina.runtime.api.Module;
+
+/**
+ * This class will hold module related utility functions.
+ *
+ * @since 2.0.0
+ */
+public class ModuleUtils {
+
+    /**
+     * File standard library package ID.
+     */
+    private static Module FILE_MODULE = null;
```

Comment:
This might break the checkstyle since you need to have constant (public static final) if you are using all caps.

Reply:
fixed